tests/unit/factories.py

```diff
@@ -50,58 +70,66 @@ class Meta:
 
 
 class MockGithubRunnerCharmUnitFactory(factory.Factory):
+    """Mock github-runner charm unit."""  # noqa: DCO060
+
     class Meta:
+        """Configuration for factory."""  # noqa: DCO060
+
         model = MagicMock
 
     name = "github-runner/0"
 
 
 class MockGithubRunnerCharmAppFactory(factory.Factory):
-    """Mock github-runner charm app."""
+    """Mock github-runner charm app."""  # noqa: DCO060
 
     class Meta:
+        """Configuration for factory."""  # noqa: DCO060
+
         model = MagicMock
 
     planned_units = 1
     name = "github-runner"
 
 
 class MockGithubRunnerCharmModelFactory(factory.Factory):
-    """Mock github-runner charm model."""
+    """Mock github-runner charm model."""  # noqa: DCO060
```

Comment:
What's a DCO060?

Reply:
This is to satisfy `flake8-docstrings-complete`.
The explanation for this error is at the top of the file - there was feedback to mention it once rather than repeating the reason for ignoring it on every line where it's used within the file :)
Would you prefer that it be on every line? 
Here's the explanation for DC060!
```
# DC060: Docstrings have been abbreviated for factories, checking for docstrings on model
# attributes can be skipped.
```